tools/topology/platform/intel/cml-sku.m4

```diff
@@ -0,0 +1,11 @@
+include(`platform/intel/cnl.m4')
```

Comment:
The naming for SKU should be according to the DAI differentiation (if any) e.g.
1) All CML boards use same SSP, DMIC then just call it cml.m4
2) Some CML boards use SSP2 and other use SSP3 then create cml-ssp2.m4 and cml-ssp3.m4
3) Same as 2 but for DMICs.